src/backend/api/Fusion.Resources.Domain/Commands/Tasks/GetActionsForRequests.cs

```diff
@@ -24,11 +26,14 @@ public GetActionsForRequests(IEnumerable<Guid> requestId)
         public class Handler : IRequestHandler<GetActionsForRequests, ILookup<Guid, QueryRequestAction>>
         {
             private readonly ResourcesDbContext db;
+            private readonly IFusionProfileResolver profileResolver;
```

Comment:
No camelCasing when naming private fields prefixed with _ ? What is convention in project? Ref. https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions